extension/src/popup/Router.tsx

```diff
@@ -248,137 +253,164 @@ export const Router = () => {
     dispatch(loadSettings());
   }, [dispatch]);
 
-  if (
+  const showNav =
+    location.pathname &&
+    ((location.pathname === "/" &&
+      applicationState === APPLICATION_STATE.MNEMONIC_PHRASE_CONFIRMED) ||
+      location.pathname === ROUTES.account ||
+      location.pathname === ROUTES.accountHistory ||
+      location.pathname === ROUTES.settings ||
+      location.pathname === ROUTES.connectWallet ||
+      location.pathname === ROUTES.connectWalletPlugin ||
+      location.pathname === ROUTES.swapSettings ||
+      location.pathname === ROUTES.sendPaymentAmount ||
```

Comment:
think it'd be a little cleaner to create an array of the ROUTES that need to showNav and an array for ROUTES that need isAppLayout. it'll be a little easier to edit a list in array form